src/components/UI/Card.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { VscEdit } from "react-icons/vsc";
+import { VscSave } from "react-icons/vsc";
+import { VscClose } from "react-icons/vsc";
+
+import "./Card.css";
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      clicked: false,
+      inputValue: "",
+      textareaValue: "",
+      input: "",
+      textarea: ""
```

Comment:
Since the data is to be passed to be used in card component, better use objects in state to represent the data - both for data itself and temp/backup values. It will aloow you to use spread operator to copy values correctly.